kdenlive/umbrel-app.yml

```diff
@@ -0,0 +1,33 @@
+manifestVersion: 1
+id: kdenlive
+category: Media
+name: KdenLive
+version: "421.12.3"
+tagline: Free and open source video editor.
+description: >-
+  Kdenlive is an open source video editor. The project was started around 2003. Kdenlive is built on Qt and the KDE Frameworks libraries. Most of the video processing is done by the MLT Framework, which relies on many other open source projects like FFmpeg, frei0r, movit, ladspa, sox, etc…
+
+  Our software was designed to answer most needs, from basic video editing to professional work.
+
+  Kdenlive is developed by a small team and new contributors are welcome.
+
+  Multi-track video editing - Kdenlive allows you to use and arrange several audio and video tracks, each one can be locked or muted to your convenience.
+  Use any audio / video format - Being based on the powerful FFmpeg libraries, Kdenlive can use almost any audio and video formats directly without the need to convert or re-encode your clips.
+  Configurable interface and shortcuts - You can arrange and save your custom interface layouts to fit your workflow.Keyboard shortcuts can also be configured to match your preferences.
+  Automatic backup - Your project file is automatically backed up every few minutes, and older versions of the project file are also kept in case you need to roll back to a previous version.
+developer:  Jean-Baptiste Mardelle, Vincent Pinon, Nicolas Carion, Simon A. Eugster
+website: https://kdenlive.org/en/
+dependencies: []
+repo: https://github.com/KDE/kdenlive
+support: https://forum.kde.org/viewforum.php?f=262
+port: 3000
+gallery:
+  - 1.jpg
+  - 2.jpg
+  - 3.jpg
+path: ""
+defaultUsername: ""
+defaultPassword: ""
+torOnly: false
+submitter: Pranshu Agrawal
+submission:
```

Comment:
Could you add the submission url, which will be the url of this PR?

Reply:
done changes for this